usermods/pov_display/usermod_pov_display.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+#include "wled.h"
+#include <PNGdec.h>
+static const char _data_FX_MODE_POV_IMAGE[] PROGMEM = "POV Image@!;;;1";
+
+PNG png;
```

Comment:
These two should be part of class (instantiation), not globals.

Reply:
This was moved accordingly to the comment